app/scripts/components/common/card.tsx

```diff
@@ -319,16 +321,17 @@ function CardComponent(props: CardComponentProps) {
     onCardClickCapture
   } = props;
 
+  const linkProps = href? { href } : {
+    as: Link,
+    to: linkTo
+  };
```

Comment:
I'd like to suggest a different implementation for this.

Instead of adding a `href` property to cards that can possible lead to some confusion, I'd propose to use the `linkTo` property even for external links.
Then, here, you'd check if the links starts with `http`. If so it is external.

```js
const linkProps = linkTo.match(/^https?:\/\//)
  ? { href: linkTo }
  : {
      as: Link,
      to: linkTo
    };
```

Reply:
Apologies if my suggestion was not clear. That check should go in the `Card` component. This component, as you said, can be used for internal links as well.

The result is that you can then use the `linkTo` property for both internal and external.

So, instead of having:
```
<Card
  ...
   href={d.external?.url}
   linkTo={getStoryPath(d)}
/>
```

you can have
```
<Card
  ...
   linkTo={d.external?.url ?? getStoryPath(d)}
/>
```
And you leave it to Card to figure out if it is internal or not.

Reply:
I meant any regex check does not seem necessary because the original design only allows the link as an external link (which always should start with https). There are some moving parts right now but I do agree that this should go as a prop for Card level.